### PR TITLE
Improve html

### PR DESCRIPTION
**Pull request by JulianRoesner**
# Short Description
We have [this ticket](https://steuerlotse.atlassian.net/browse/STL-1224?atlOrigin=eyJpIjoiNTNjZWY2NTUzZGUxNDY1NGIxNjliZmMyNWY2MTVhZDEiLCJwIjoiaiJ9) to check whether there are any invalid parts in our html. 
I have adjusted the html a little to fix some of the problems but not all of them. 

# Changes
- Add alt attribute to images where it was missing
- Correctly set width without "px"
- Correct aria-controls where incorrect ids were used 
- Add `role=button` to divs that are buttons
- Replace example_input with data-example-input
- Replace detail attribute with data-detail
- Correct aria-labeledby attribute

# Feedback
- Do you see any problems with that

### CODE REVIEW

File: webapp/app/forms/steps/lotse_multistep_flow_steps/personal_data_steps.py [hunk 1]
```diff
@@ -40,15 +40,15 @@ class Form(SteuerlotseBaseForm):
             validators=())
         familienstand_married_lived_separated = YesNoField(
             label=_l('form.lotse.familienstand_married_lived_separated'),
-            render_kw={'example_input': _l('form.lotse.familienstand_married_lived_separated.example_input'),
+            render_kw={'data-example-input': _l('form.lotse.familienstand_married_lived_separated.example_input'),
```
Comment (frederike-ramin) [inline webapp/app/forms/steps/lotse_multistep_flow_steps/personal_data_steps.py:43]: Could you maybe explain what problem this fixes?

**Review by frederike-ramin — APPROVED** [review]
Nice, I'm glad you explored that. :)
I just have one question.

File: webapp/app/forms/steps/lotse_multistep_flow_steps/personal_data_steps.py (hunk 1 above)
Comment (JulianRoesner) [reply webapp/app/forms/steps/lotse_multistep_flow_steps/personal_data_steps.py:43]: The render_kw are also set as attributes on the HTML element. Therefore this becomes the `attribute` example_input which is not a valid attribute. By using [a data-attribute](https://developer.mozilla.org/en-US/docs/Learn/HTML/Howto/Use_data_attributes) for that, we get rid of this problem.

Comment (frederike-ramin) [reply webapp/app/forms/steps/lotse_multistep_flow_steps/personal_data_steps.py:43]: Nice! Lovin' it!

**Pull request merged**